contract/data/mail_template.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" ?>
+<odoo noupdate="1">
+
+    <record id="email_contract_template" model="mail.template">
+        <field name="name">Email Contract Template</field>
+        <field name="email_from">${(object.user_id.email and '%s &lt;%s&gt;' % (object.user_id.name, object.user_id.email) or '')|safe}</field>
+        <field name="subject">${object.company_id.name} Contract (Ref ${object.name or 'n/a'})</field>
+        <field name="partner_to">${object.partner_id.id}</field>
+        <field name="model_id" ref="contract.model_account_analytic_account"/>
+        <field name="auto_delete" eval="True"/>
+        <field name="report_template" ref="contract.report_contract"/>
+        <field name="report_name">Contract</field>
+        <field name="lang">${object.partner_id.lang}</field>
+        <field name="body_html"><![CDATA[
+<div style="font-family: 'Lucida Grande', Ubuntu, Arial, Verdana, sans-serif; font-size: 12px; color: rgb(34, 34, 34); background-color: #FFF; ">
+    <p>Hello ${object.partner_id.name or ''},</p>
+    <p>A new contract has been created: </p>
+
+    <p style="border-left: 1px solid #8e0000; margin-left: 30px;">
+       &nbsp;&nbsp;<strong>REFERENCES</strong><br />
+       &nbsp;&nbsp;Contract: <strong>${object.name}</strong><br />
+       &nbsp;&nbsp;Contract date start: ${object.date_start or ''}<br />
+
+       % if object.user_id:
+       &nbsp;&nbsp;Your contact: <a href="mailto:${object.user_id.email or ''}?subject=Contract%20${object.name}">${object.user_id.name}</a>
```

Comment:
s/Your contact/Your Contact

Also- it would be best to not have this be a link if the user does not have an email.